index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <!--Hero Section-->
+  <section id="hero">
+
+    <header>
+      <nav id="navbar">
+        <div class="hamburger">
+          <a href="#"><i class="fa fa-bars"></i></a>
+        </div>
+      </nav>
+    </header>
+
+    <div class="banner">
+
+      <div class="introduction">
+
+        <div class="title">
+          <div class="self">Hey there.</div>
+          <div class="self">I'm Alex.</div>
+          <div class="profession">I’m a software developer</div>
```

Comment:
Great work done in this section 💪🥳 but we need to use hx tags for titles this is for semantic reasons as well as for better accessibility